lib/routes/tongji/sem/_utils.ts

```diff
@@ -0,0 +1,69 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export async function getNotifByPage(pageNumber: number) {
+    const pageUrl: string = `https://sem.tongji.edu.cn/semch/category/frontpage/notice/page/${pageNumber}`;
+
+    try {
+        const response = await got.get(pageUrl, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.101.76 Safari/537.36',
+            },
+        });
+
+        const html = response.body;
+        const $ = load(html);
+
+        const notifListElements = $('#page-wrap > div.maim_pages > div > div.leftmain_page > div > ul > li');
+
+        return notifListElements.toArray().map((Element) => {
+            const aTagFirst = $(Element).find('a.bt');
+            const aTagSecond = $(Element).find('a.time');
+
+            const title = aTagFirst.attr('title');
+            const href = aTagFirst.attr('href');
+            const time = aTagSecond.text().trim();
+
+            return {
+                title,
+                link: href,
+                pubDate: parseDate(time, 'YYYY-MM-DD'),
+            };
+        });
+    } catch {
+        // console.error(error);
+    }
+    return [];
+}
+
+export async function getLastPageNumber() {
+    try {
+        const response = await got.get('https://sem.tongji.edu.cn/semch/category/frontpage/notice', {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.101.76 Safari/537.36',
```

Comment:
RSSHub will use a randomised user agent of [Chrome on mac](https://github.com/DIYgod/RSSHub/blob/263d6b818ea551e9a498207ba1f0a912b6370224/lib/config.ts#L344) by default. Does the site only works with this fixed old version of Chrome on Windows?

Reply:
I've fixed it.